app/react/Library/helpers/libraryFilters.js

```diff
@@ -87,14 +87,24 @@ export function parseWithAggregations(filters, aggregations, showNoValue = true)
       property.options = propertyAggregations.buckets
         .map(normalizeBucket)
         .filter(opt => opt.results || (!showNoValue && opt.value === 'missing'));
-
-      property.totalPossibleOptions = propertyAggregations.count;
     }
 
     return property;
   });
 }
 
+export const prepareDefaultFilters = fields =>
+  fields.map(field => {
+    if (!field.options) {
+      return field;
+    }
+    if (!field.defaultfilter) {
+      return field;
+    }
+    const filteredOptions = field.options.filter(option => option.id !== 'missing');
```

Comment:
I think this could be a single condition

Reply:
Yes, I totally forgot to refactor that!